src/posthog-core.ts

```diff
@@ -124,7 +124,7 @@ export const defaultConfig = (): PostHogConfig => ({
     save_referrer: true,
     capture_pageview: true,
     capture_pageleave: true, // We'll only capture pageleave events if capture_pageview is also true
-    debug: false,
+    debug: (location?.search && location.search.indexOf('__posthog_debug=true') !== -1) || false,
```

Comment:
Ugly because we have to support IE11, so no URLSearchParams

Reply:
surprised we don't already have a convenience function for is blah in the URL but let's not overcomplicate by adding one if we don't already 😊

Reply:
We do have `_isString` so you could really belt and braces by  replacing `location?.search` with `_isString(location?.search)` but that's very nit picky and I _don't think_ it adds enough safety to the check to care